packages/rancher-wins-upgrader/charts/values.yaml

```diff
@@ -0,0 +1,40 @@
+# Default values for rancher-windows-exporter.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+# Configuration
+
+global:
+  cattle:
+    systemDefaultRegistry: ""
+
+prefixPath: c:\\
+
+## Contains one or more configurations for the wins server to be applied across all of the nodes based on the nodeSelector and tolerations provided
+##
+winsConfigs:
+- name: default
+  image:
+    # TODO(aiyengar2): replace with an image that just contains wins
+    repository: rancher/wmi_exporter-package
```

Comment:
We don't currently have a Dockerfile that tracks different packaged versions of wins, but this needs to be created. 

This should pretty much be `https://github.com/rancher/wmi_exporter-package/blob/master/package/windows/Dockerfile` + `https://github.com/rancher/wmi_exporter-package/blob/master/package/windows/Dockerfile.20H2` maintained in the `rancher/wins` repository after dropping the `$BINARY_URL` + `COPY --from=download /wmi-exporter.exe /etc/wmi-exporter/` steps from it and the final command.

We'll also need to copy over https://github.com/rancher/wmi_exporter-package/blob/master/.drone.yml.

@luthermonson do you have any thoughts about this?